Assets/Scripts/EncounterScripts/Encounter.cs

```diff
@@ -0,0 +1,12 @@
+using UnityEngine;
+
+public class Encounter : MonoBehaviour 
+{
+	[SerializeField]
+	private GameObject[] enemyGroup;
+
+	public void StartEncounter()
+	{
+		Debug.Log("StartEncounter");
```

Comment:
Use the sceneGod for this
example:  SceneGod.SInstance.EnterCombatState(enemyGroup);